tests/test-infra/azure-aks.bicep

```diff
@@ -156,7 +156,7 @@ resource aks 'Microsoft.ContainerService/managedClusters@2021-07-01' = {
   tags: {}
   sku: {
     name: 'Basic'
-    tier: 'Free'
+    tier: 'Paid'
```

Comment:
I am curious about why it caused failures.

Reply:
I am curious too!!

First, this is NOT a Dapr issue. The issue is with the K8s control plane.

I spent about 2 weeks debugging this with the AKS team and this was the result.

At a high level, the "paid" control plane has better HA (control plane resources are replicated). More relevant to us, in the "free" tier the nodes connect to the control plane via a  sort of P2P VPN which we've seen sometimes can have glitches. In the "paid" tier, the VPN is different.

Reply:
That sounds reasonable to me, the network issue caused failures.